packages/client/wallets/aa/src/CrossmintAASDK.ts

```diff
@@ -142,4 +146,22 @@ export class CrossmintAASDK {
             localStorage.removeItem(key);
         });
     }
+
+    private getEntryPointAddress(entryPointVersion: EntryPointVersion): EntryPoint {
+        return entryPointVersion === 0.6 ? ENTRYPOINT_ADDRESS_V06 : ENTRYPOINT_ADDRESS_V07;
+    }
```

Comment:
As this is one line, I don't see the value of splitting this in two functions, which implies having to call both instead of one